src/containers/DeveloperPortal/DocumentationSitemap/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import {A, Container} from 'components';
+import './DocumentationSitemap.scss';
+
+// TODO: update links when ready
+export default function DocumentationSitemap() {
+  return (
+    <div className="DocumentationSitemap">
+      <Container className="DocumentationSitemap__container">
+        <div className="DocumentationSitemap__title">Documentation Sitemap</div>
+        <div className="DocumentationSitemap__topics">
+          <div className="DocumentationSitemap__topic-container">
+            <div className="DocumentationSitemap__topic-header">Living Whitepaper</div>
+            <A href="/" newWindow={false} className="DocumentationSitemap__link">
```

Comment:
😬 props are not in alphabetical order, I think we've followed that everywhere

Reply:
![alphabetize](https://user-images.githubusercontent.com/5943963/124196875-e6f79e00-da92-11eb-83a1-ef295b81564f.jpeg)